### PR TITLE
Match NTSC z_parameter.c and z_construct.c

### PR DESCRIPTION
**Pull request by cadmic**
Seems like they ran out of REG space and had to do some reshuffling

### CODE REVIEW

File: src/code/z_kaleido_setup.c [hunk 1]
```diff
@@ -84,7 +84,7 @@ void KaleidoSetup_Update(PlayState* play) {
             pauseCtx->mainState = PAUSE_MAIN_STATE_SWITCHING_PAGE;
 
             //! @bug REG collision
-            if (R_START_LABEL_DD(0) == 0) {
+            if (ZREG(48) == 0) {
```
Comment (Dragorn421) [inline src/code/z_kaleido_setup.c:87]: Can you add to the comment which regs this is confonded with

afaict R_START_LABEL_SCALE for NTSC and R_START_LABEL_DD(0) for PAL

**Review by Dragorn421 — COMMENTED** [review]
Ouch

**Review by Dragorn421 — COMMENTED** [review]
I'd like to see the NTSC regs also used in `Regs_InitDataImpl`. To elaborate:

<details>

e.g. with `R_B_LABEL_SCALE(0)` aka `WREG(8)` (NTSC) or `WREG(37)` (PAL)

matching PAL z_construct.c `Regs_InitDataImpl` excerpt:
```c
    WREG(8) = 100;
...
    WREG(37) = 100;
```

aka, still matching PAL:
```c
    WREG(8) = 100; // R_B_LABEL_SCALE(0) under OOT_NTSC
...
    R_B_LABEL_SCALE(0) = 100; // WREG(37) under OOT_PAL
```

(To be clear we're thinking the `WREG(8) =` is just leftovers from NTSC that wasn't deleted.)

To match both this PR does:

```c
    WREG(8) = 100; // R_B_LABEL_SCALE(0) under OOT_NTSC
...
#if OOT_PAL
    R_B_LABEL_SCALE(0) = 100;
#endif
```

I would additionally ifdef around WREG(8) too like:

```c
#if OOT_NTSC
    R_B_LABEL_SCALE(0) = 100;
#else
    // OOT_PAL
    // Remnants from NTSC versions. This corresponds to expanding the NTSC counterpart above to `?REG()` macros under OOT_NTSC.
    WREG(8) = 100; // R_B_LABEL_SCALE(0) for NTSC
#endif
...
#if OOT_PAL
    R_B_LABEL_SCALE(0) = 100;
#endif
```

(hard to write the comment cleanly idk)

</details>

**Comment by cadmic** [comment]
Yeah I see what you mean, how's this?

**Review by Dragorn421 — APPROVED** [review]
:+1: 

`R_B_LABEL_SCALE` only being used for dsdx / dtdy is sus (it should also be involved in screen space width/height of the rectangle), but well jank code be jank

**Review by fig02 — APPROVED** [review]

**Pull request merged**